po/zh.po

```diff
@@ -929,42 +929,42 @@ msgstr "没有找到操作"
 #: src/formaction.cpp:333
 #, fuzzy, c-format
 msgid "Not a command: %s"
-msgstr "未知的命令 `%s' "
+msgstr "未知的命令：%s"
 
 #: src/formaction.cpp:509
 msgid "Saving bookmark..."
-msgstr "保存书签..."
+msgstr "保存书签......"
 
 #: src/formaction.cpp:515 src/formaction.cpp:582
 msgid "Saved bookmark."
-msgstr "已保存的书签."
+msgstr "已保存的书签。"
 
 #: src/formaction.cpp:518 src/formaction.cpp:585
 msgid "Error while saving bookmark: "
-msgstr "当保存书签时出错: "
+msgstr "当保存书签时出错："
 
 #: src/formaction.cpp:553
 msgid "URL: "
-msgstr "链接: "
+msgstr "链接："
 
 #: src/formaction.cpp:560
 msgid "Description: "
-msgstr "描述: "
+msgstr "描述："
 
 #: src/formaction.cpp:561
 msgid "Feed title: "
-msgstr "种子标题: "
+msgstr "种子标题："
 
 #: src/formaction.cpp:576
 #, fuzzy
 msgid "Saving bookmark on autopilot..."
-msgstr "保存书签..."
+msgstr "保存书签......"
```

Comment:
Still fuzzy?

Reply:
Fuzzy items remain since this PR only focuses on styles. Same as below.

Reply:
I see.